lib/game.rb

```diff
@@ -1,8 +1,9 @@
 class Game
-  attr_reader :season, :away_goals, :home_goals
+  attr_reader :season, :away_goals, :home_goals, :game_id
```

Comment:
Great job on adding those commas before the final attribute. They cause problems when they are missing.

Reply:
yeah i've definitely had issues with the commas in the past... oops lol

Reply:
The terminal will always expose the weakest link in your code.